### ISSUE TITLE
[CAIA A11y Research - PILOT] Authentication Experience, My VA (dot indicator usage)

### ISSUE THREAD

**Issue by sara-amanda** [department-of-veterans-affairs/va.gov-team#75143]
> [!IMPORTANT]
> - **TICKET STATUS:** Moved from `Blocked` to `READY` on 2/12/2024
> - **PILOT RESCHEDULED:** ~2/20/2024 1-2 p.m. ET~ 2/23/2024 at 3:30 p.m. 
> - cc: @coforma-jamie @EvanAtCoforma 

# CAIA A11y Support
This is a child ticket of the research ticket [73932](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73932) - created to track work, completed from this pilot session.
> [!TIP]
> **DEFINITION OF DONE**: Refer to the task list in this ticket, for actionable items.

## CAIA Assignee(s)

<details><summary>Toggle to view CAIA info</summary>

- @coforma-jamie 
- @sara-amanda 

</details>

## Team Info

<details><summary>Toggle to view team info</summary>

- Team name:Authentication Experience (My VA)
- OCTO-DE product owner: Samara Strauss
- Product manager: Anastasia(AJ) Jakabcin
- Designer: Matt Marino
- FE engineer: Allison Liu
- Product/team Slack channel: [#accountexp-authexp](https://dsva.slack.com/archives/C909ZG2BB)

</details>

## Description

<details><summary>Toggle to view the description</summary>

A CAIA a11y will participate as a user in a pilot of the [Authenticated Experience, My VA (dot indicator usage) research study](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73932).

- **Researcher**: The VFS team researcher will test their prototype and conversation guide by conducting the pilot session with a CAIA A11Y.
- **User**: The CAIA A11Y team member will act as a real user in the pilot session.
- **Assistive Tech**: The CAIA A11Y team member will follow the prompts given by the VFS Researcher, while using and demonstrating how the behavior of a `screen reader` would behave.

### Pilot Session

- **~CONFIRMED~**: ~Friday, Feb. 9, 2024~ 
- ~BEING RESCHEDULED for 2/16 or the following week, per Matt on 2/12/2024.~  
- **RESCHEDULED** **New Pilot Date:** 2/23/2024 3:30 p.m. ET (on CAIA Capacity Calendar)
    - [Reached out to the team via Slack to field interest](https://oddball.slack.com/archives/C05LB36ATAQ/p1706805440895549) 
    - Jamie will be representing CAIA a11ys
- **Prototype**: CodePen - link the week of 2/5
- **Research Convo Guide**: [Auth Exp. My VA Dot indicator Usage)](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/onsite-notifications/dot-indicator/research/dot-indicator-convo-guide-for-screen-readers.md)

</details>

### Assistive Tech Options

> [!IMPORTANT]
> **DEVICE SUPPORT DURING RESEARCH SESSIONS**
> - **Available AT**: `Screen readers` on a `desktop` device is currently the primary source of AT that CAIA A11ys are able to support, in addition to `screen magnification` for user research sessions.
> - **Additional Specifications:** If there is a particular `test user`, `browser` or `operating system` you would like us to use, please let us know in advance via this ticket's comments, tagging the assignee - or reach out to @coforma-terry in the CAIA Slack Channel: [#sitewide-content-accessibility-ia ](https://dsva.slack.com/archives/C01K37HRUAH)

## Definition of Done

```[tasklist]
### A11y Tasks
- [x] **Schedule CAIA a11y:** Eli or Jamie
- [x] **Determine date and time:** ~2/9/24, TIME 10:15 a.m.~ `BEING RESCHEDULED` (Likely Week of 2/12/2024) 2/23
- [x] **Calendar** Add to a11y Capacity Calendar
- [x] **RESCHEDULE PILOT** When Matt is back in the office (Likely week of 2/12/2024) `Pending New Date/Time` Update as of 2/12/2024: Matt can do 2/16/2024 or the following week, pending CAIA's availability. **NEW DATE AND TIME CONFIRMED:** 2/23/2024 3:30 p.m. ET
- [x] **Update CAIA A11y Capacity Calendar:** Add new invite for new pilot, when scheduled
- [x] **Review research** [conversation guide](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/identity-personalization/onsite-notifications/dot-indicator/research/dot-indicator-convo-guide-for-screen-readers.md) and provide feedback in [comments](https://github.com/department-of-veterans-affairs/va.gov-team/issues/75143#issuecomment-1922221752)
- [x] [Request AT device and tech to be used in sessions](https://github.com/department-of-veterans-affairs/va.gov-team/issues/75143#:~:text=DEVICE%20SUPPORT%20DURING%20RESEARCH%20SESSIONS)
- [x] [Receive AT device and tech from Auth Exp Team](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73932#:~:text=Device%20%26%20AT%20Combinations,Table%202%3A%20Technology) to be used in sessions to prep for pilot
- [x] **Review prototype** in CodePen - `PENDING PROTOTYPE`
- [x] **Run pilot** with VFS Researcher - no longer taking place on 2/9/2024 `Pending New Date/Time` (likely 2/16 or the following week) **NEW PILOT DATE AND TIME:** 2/23/2024 3:30 p.m. ET
- [x] **Take notes** in [GH comment](https://github.com/department-of-veterans-affairs/va.gov-team/issues/75143#issuecomment-1962014259)
- [ ] **Follow-up meeting** with team to review prototype edits: 2/28/24
```

**Comment by sara-amanda** [comment]
## Pilot Prep

@mattmarino-adhoc 

- Do you happen to know [the types of Assistive Tech](https://github.com/department-of-veterans-affairs/va.gov-team/issues/75143#:~:text=DEVICE%20SUPPORT%20DURING%20RESEARCH%20SESSIONS) that will be used in the research sessions?
- [The device and AT information](https://github.com/department-of-veterans-affairs/va.gov-team/issues/73932#:~:text=Device%20%26%20AT%20Combinations,Table%202%3A%20Technology) can help the team prep for your pilot. We know this information is not always immediately available from Perigean. 

cc: @coforma-terry

**Comment by sara-amanda** [comment]
## Pilot Postponed
- The pilot session scheduled for 2/9/2024 **will not be taking place**. 
- The pilot session **will be rescheduled** with Matt, when he is back in the office. 
- Tentative for next week, date and time `TBD`. 
- _Updates will be made in this Pilot Ticket._ 

cc: @mattmarino-adhoc @coforma-terry @eli-oat @coforma-jamie @andaleliz

**Comment by coforma-jamie** [comment]
@mattmarino-adhoc couple of notes from today:
- only show "two dots with badge indicator" (don't bother with the "two dots" prototype)
- add href="#" or a dummy link to all links
- ask how the participant would like to receive the links - chat, email, or read to them
- the badge indicator currently doesn't announce the number '2' to assistive tech

I'll keep this ticket open, and close it after our meeting next week! :)
